benchmark/galsim/Sandbox.jl

```diff
@@ -0,0 +1,24 @@
+module Sandbox
```

Comment:
There's a known issue where ForwardDiff has to be loaded before StaticArrays. I wonder if some that's what's happening when you use Sandbox to workaround the bug. Hopefully the bug will just go away as Julia 0.6 matures.